src/components/include/cos_kernel_api.h

```diff
@@ -30,7 +30,7 @@ struct cos_meminfo {
 /* Component captbl/pgtbl allocation information */
 struct cos_compinfo {
 	/* capabilities to higher-order capability tables (or -1) */
-	capid_t pgtbl_cap, captbl_cap, comp_cap;
+	capid_t pgtbl_cap, captbl_cap, comp_cap, local_pgtbl_cap;
```

Comment:
I would think that `pgtbl_cap` will still be the local one.  `global_pgtbl_cap` should be the new one.  I'm just thinking that the common case shared with other parts of the system will still be the normal local operations.  We want to use the global ones intentionally, so they should be the exceptional case.  What do you think?